packages/twenty-emails/src/components/WhatIsTwenty.tsx

```diff
@@ -11,35 +10,7 @@ export const WhatIsTwenty = () => {
         It's a CRM, a software to help businesses manage their customer data and
         relationships efficiently.
       </MainText>
-      <Row>
-        <Column>
-          <ShadowText>
-            <Link href="https://twenty.com/" value="Website" />
-          </ShadowText>
-        </Column>
-        <Column>
-          <ShadowText>
-            <Link href="https://github.com/twentyhq/twenty" value="Github" />
-          </ShadowText>
-        </Column>
-        <Column>
-          <ShadowText>
-            <Link href="https://twenty.com/user-guide" value="User guide" />
-          </ShadowText>
-        </Column>
-        <Column>
-          <ShadowText>
-            <Link href="https://docs.twenty.com/" value="Developers" />
-          </ShadowText>
-        </Column>
-      </Row>
-      <ShadowText>
-        Twenty.com Public Benefit Corporation
-        <br />
-        2261 Market Street #5275
-        <br />
-        San Francisco, CA 94114
-      </ShadowText>
+      <Footer />
```

Comment:
Good idea. We should probably add this to every email as a BaseFooter like the base header?

Reply:
There are some emails (as seen in Figma) that don't have footer links in them:

![alt text](<Screenshot 2025-01-06 at 10.14.35 AM.png>)

However, the Footer is already being used in all the places it was before as it's there as a default in <WhatIsTwenty/>. Should I extract it outside of it and add <Footer /> to all the places <WhatIsTwenty /> was being used?

Reply:
Good point. But I think we'll want a footer on every emails for consistency (e.g. one day we'll need to add "manage your preferences" button). We might edit the footer in a followup issue to make it smaller and better suited for all emails, but let's include it in all emails for now

cc @Bonapara let me know if you disagree

Reply:
Opened #9622 for this. Resolving this conversation.